d2js/d2wasm/types.go

```diff
@@ -33,19 +33,29 @@ type BoardPositionResponse struct {
 
 type CompileRequest struct {
 	FS   map[string]string `json:"fs"`
-	Opts *RenderOptions    `json:"options"`
+	Opts *CompileOptions   `json:"options"`
 }
 
 type RenderOptions struct {
-	Layout  *string `json:"layout"`
-	Sketch  *bool   `json:"sketch"`
-	ThemeID *int64  `json:"themeID"`
+	Pad           *int64   `json:"pad"`
+	Sketch        *bool    `json:"sketch"`
+	Center        *bool    `json:"center"`
+	ThemeID       *int64   `json:"themeID"`
+	DarkThemeID   *int64   `json:"darkThemeID"`
+	Scale         *float64 `json:"scale"`
+	ForceAppendix *bool    `json:"forceAppendix"`
+}
+
+type CompileOptions struct {
+	RenderOptions
+	Layout *string `json:"layout"`
 }
 
 type CompileResponse struct {
 	FS      map[string]string `json:"fs"`
 	Diagram d2target.Diagram  `json:"diagram"`
 	Graph   d2graph.Graph     `json:"graph"`
+	Options RenderOptions     `json:"options"`
```

Comment:
We should still name this RenderOptions -- it's confusing to get `options` back from a response after sending `options` in the request and have it be a different set